front/components/sparkle/ThemeContext.tsx

```diff
@@ -1,6 +1,6 @@
 import { createContext, useContext, useEffect, useState } from "react";
 
-export type Theme = "dark" | "light" | "system";
+type Theme = "dark" | "light" | "system";
```

Comment:
Unrelated but did not need an export as not used elsewhere (code was added this morning)